### PR TITLE
chore: update node version to 16 for custom domain function and CI

### PR DESCRIPTION
**Pull request by dannyrandall**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

**Review by efekarakus — APPROVED** [review]
Thank you!

**Review by iamhopaul123 — APPROVED** [review]
Probably wait until the e2e test fix gets merged.

**Pull request merged**
